src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
```

Comment:
Regarding `Lottery` class i have some doughts that we need that just for purpose to get random ball. If we need random ball we can create some Utility class for that purpose or even Service.
The Lottery should as seems to me have some list of balls, unique number, created time and validTo time like 
![image](https://github.com/khshanovskyi/my-lottery-task-review/assets/55089853/9bad03ca-1588-4d7e-9c07-c334137e2709)
and final result will look like 
![image](https://github.com/khshanovskyi/my-lottery-task-review/assets/55089853/4c50a64b-24b9-489c-8ae5-ceb1df6a15d3)
or we can create some Lottery factory that will do that,
but the main idea that we either don't need this class and need move to utils or we need that as `Lottery` entity